specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -8572,6 +8572,7 @@
       "description": "A dictionary of http request headers."
     },
     "AmlSkill": {
+      "x-ms-client-name": "AzureMachineLearningSkill",
```

Comment:
This is a breaking change and is why the CI is flagging it.  Is this intentional?

Reply:
Yes, this is an approved client-side name change for a new preview feature being added.

Reply:
As I understand, this has no effect on the service itself.

Reply:
Correct.  It changes the type name in the generated SDKs.